src/initializeCacheClient.js

```diff
@@ -0,0 +1,35 @@
+/* Memcached cache client. */
+import { get } from 'lodash';
+import { bind, getContainer, getMetadata } from '@globality/nodule-config';
+
+import Cache from './memcached-promisify';
+
+function createCacheClient() {
+    const { config } = getContainer();
+    const memcachedConfig = get(config, 'cache.memcached');
+    const cacheHost = get(config, 'cache.hosts', '').split(',');
```

Comment:
Can we use `cache.memcached.cacheHost` instead?

Reply:
it'll interfere with the previous line

Reply:
unless - we don't need two lines

Reply:
moved the split to the Cache constructor, and put the config into cache.memcached